ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -45,6 +45,7 @@ Build_Tool_Packages:
   - perl-GD
   - perl-libwww-perl
   - perl-Time-HiRes
+  - openssh-clients
```

Comment:
Can you add a comment saying `Required by IBM to clone over ssh` or words to that effect so we remember why it's there (I can't easily get the comment lined up correctly by making a "suggested change" in the github online editor so I'm not going to try!)